src/index.ts

```diff
@@ -12,8 +12,13 @@ import * as momemt from 'moment';
 
 import { exec } from 'child_process';
 import { ChoiceType, prompt } from 'inquirer';
-import { AuthResponse, AzureEnvironment, DeviceTokenCredentials, DeviceTokenCredentialsOptions,
-    LinkedSubscription, InteractiveLoginOptions, interactiveLoginWithAuthResponse } from 'ms-rest-azure';
+import {
+    AuthResponse, AzureEnvironment, AzureTokenCredentialsOptions, DeviceTokenCredentials, DeviceTokenCredentialsOptions,
```

Comment:
nit: put them on single line

Reply:
It is too long in a single line and can not pass tslint check.

Reply:
I meant put each import on a new line.
```
AuthResponse,
AzureEnvironment,
...
```